src/options/options_handler.cpp

```diff
@@ -1368,67 +1370,96 @@ void OptionsHandler::notifySetPrintExprTypes(std::string option) {
 
 
 // main/options_handlers.h
+
+static void print_config (const char * str, std::string config) {
+  std::string s(str);
+  unsigned sz = 14;
+  if (s.size() < sz) s.resize(sz, ' ');
+  std::cout << s << ": " << config << std::endl;
+}
+
+static void print_config_cond (const char * str, bool cond = false) {
+  print_config(str, cond ? "yes" : "no");
+}
+
+void OptionsHandler::copyright(std::string option) {
```

Comment:
Why is the option argument to this function unused?

Reply:
This is due to the fact that all option handlers are required to have the same interface. Some, e.g., https://github.com/aniemetz/CVC4/blob/6ee05018780ec77c9d1f3790f4013803308e2b83/src/options/options_handler.cpp#L93, need this argument for error handling.